### ISSUE TITLE
radulf the elder stuck in building

### ISSUE THREAD

**Issue by Rotbash21** [WarEmu/WarBugs#20364]
radulf the elder for T4 epic quest is stuck in a building and unable to be interacted with

**Comment by xpander69** [comment]
Any screenshots of it?

**Comment by Rubyeclipse-war** [comment]
@Rotbash21  are you still seeing this issue / do you have any screenshots of this like xpander mentioned?

I checked on my side and I'm able to see this NPC and interact with them normally: 

![image](https://user-images.githubusercontent.com/60584990/189399158-56a345ac-b017-4b32-8dee-402698085d22.png)

**Comment by Rubyeclipse-war** [comment]
I've looked into this and made some adjustments to this NPC which should help.

Previously, this quest giver was able to be attacked by Order players, which might result in him moving and getting stuck in terrain. I've changed him to be unattackable and neutral to Order, so he should stay happily on his rock moving forward.

This will hopefully fix the issue for good, but if it ever crops up again, just let us know. Thanks!
